nfs/src/rpc_task.cpp

```diff
@@ -184,6 +207,180 @@ static void lookup_callback(
     }
 }
 
+#if 0
+static void flush_callback(
+    struct rpc_context* /* rpc */,
+    int rpc_status,
+    void *data,
+    void *private_data)
+{
```

Comment:
why have you kept it commented?
other than the name difference is it different from write_flush_callback()?
were you originally thinking of keeping a separate callback for flush and write and then you ended up using one?

Reply:
Removed the commented code.